packages/core-generators/src/renderers/typescript/templates/types.ts

```
@@ -26,37 +114,28 @@ export const tsTemplateFileMetadataSchema =
      */
     projectExportsOnly: z.boolean().optional(),
   });
 
-export type TsTemplateFileMetadata = z.infer<
-  typeof tsTemplateFileMetadataSchema
+export type TsTemplateOutputTemplateMetadata = z.infer<
+  typeof tsTemplateOutputTemplateMetadataSchema
 >;
 
-export interface TsTemplateVariable {}
+export interface TsTemplateFileVariable {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)